app/messaging/create-msg.js

```diff
@@ -25,7 +25,7 @@ function getDesirabilityAnswers (request) {
   try {
     let val = {}
     const projectSubject = getYarValue(request, 'projectSubject')
-    if (projectSubject === 'Robotics and Innovation') {
+    if (projectSubject === 'Robotics and automatic technology') {
```

Comment:
Can we revert the changes in this file for now pls? Updating scoring in another PR but it will take some time